pkg/consts/consts.go

```diff
@@ -0,0 +1,8 @@
+package consts
```

Comment:
I don't want to open Pandora's box, but to be honest, it was really convenient for me to have "consts" package, instead of "v1alpha1" which I really hate from k8s :) 

It was so obvious for me when I was using it reporter and scheduler, in which package I have defined such consts. I also read one more time: https://blog.golang.org/package-names where you can find section:
> Bad package names:
> Avoid meaningless package names. 
It does not apply to our case IN MY HUMBLE opinion, because package name is not like util, misc etc.

Reply:
Moved to v1alpha1 - file labels.go (generatedName is not there anymore)